### DISCUSSION TITLE
Sondage (lien externe)

### DISCUSSION THREAD

**Discussion by GigMatt** [MTESSDev/FRW#26] [Annonces]
Bonjour, nous pourrions avoir besoin de l'appréciation de nos clients en lien avec leur demande en ligne. 

Pour se faire, un lien externe rédigera le client à la fin du formulaire Web. 

Questions:
- Est-ce possible d'appliquer ce type de lien lorsque nous terminons la demande FRW? 
- Pouvons-nous placer ce lien dans la section validation, par exemple?

Je vous remercie pour votre précieuse collaboration.

**Comment by j0ber** [comment]
Bonjour,

Le meilleur endroit pour effectuer ce que vous voulez serait dans la page de confirmation qui est affichée après la soumission.
Il est possible d'ajouter du texte  (ou html) après l'avis de confirmation, c'est configurable.

La question était très pertinente, nous n'avions pas d'exemple à ce sujet.

J'ai ajouté un exemple dans notre guide configuration des formulaires qui devrait répondre à votre besoin.

[Voir l'exemple](https://formulaires.it.mtess.gouv.qc.ca/Form/7/P700U/0/N/#p=pageConfirmation)

**Reply by GigMatt** [reply]
C'est un excellent service, merci.
